docs/_docs/installation/macos.md

```diff
@@ -13,7 +13,7 @@ xcode-select --install
 ## Install Ruby
 
 Jekyll requires **Ruby v{{ site.data.ruby.min_version }}** or higher.
-macOS Catalina 10.15 ships with Ruby 2.6.3. Check your Ruby version using `ruby -v`.
+macOS Big Sur 11.2 ships with Ruby 2.6.3. Check your Ruby version using `ruby -v`.
```

Comment:
Shouldn't this be referencing 11.0? I haven't upgraded yet, but if this is replacing 10.15 in the text, it seems appropriate to use the "x.0" release as the baseline for requirements.

Has the version of Ruby been upgraded in any of the point releases?

Reply:
Hi @chrisfinazzo , that's a good question. I'm not sure if the minor releases have changed Ruby. I just used the latest minor version, since that's what was there before (i.e. 10.15 => 11.2).

Reply:
```suggestion
macOS Big Sur 11.x ships with Ruby 2.6.3. Check your Ruby version using `ruby -v`.
```